SparkPost/2web2ui
# FE-690 - Merges several greenkeepers PRs into one

### What Changed
 - Merged all gc prs labeled with `awaiting-pr-merge` except babel7 (#891)  https://github.com/SparkPost/2web2ui/labels/awaiting-pr-merge 

### How To Test 
 - Development
   - npm install
   - npm start
   - Use app normally
 - Prod Build
   -  Modify `apiBase` in `config/default.js` to use dev apiBase
   - cd build
   - npx serve
   - http://localhost:5000
   - You should be able to keep it using normally
 - Test
   - Make sure all tests pass locally
   - Make sure all tests pass in CI
